website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
+    <%include file="../alert.mako"/>
```

Comment:
Oh, sorry for my misunderstanding! Basically, there is an assumption that node object on mako context is only used in project page. So for when node is not defined, it shows the alert from base.mako.